openapi-specs/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.1.0
 info:
   title: Open Payments
-  version: '1.0'
+  version: "1.0"
```

Comment:
Are these formatting changes intentional?
This seems to be the only file using double quotes.
I'm trying to see if we can make `spectral-cli` enforce formatting for us.
https://meta.stoplight.io/docs/spectral/01baf06bdd05a-rulesets
This just suggests using prettier (along with spectral):
https://github.com/Nexmo/api-specification#check-your-api-spec-for-validity-and-style

Reply:
No, not intentional. I'll try to revert that.

Reply:
Enforcing would be great. But like you said, it may help to just include a `prettierrc.yml`.

Reply:
`shared/schemas.yaml` is also double quoted